localization/l10n/bundle.l10n.json

```diff
@@ -238,6 +238,16 @@
   "Expand All": "Expand All",
   "Collapse All": "Collapse All",
   "Filter for any field...": "Filter for any field...",
+  "Microsoft would like your feedback": "Microsoft would like your feedback",
+  "Overall, how satisfied are you with the MSSQL extension?": "Overall, how satisfied are you with the MSSQL extension?",
+  "Very Satisfied": "Very Satisfied",
+  "Satisfied": "Satisfied",
+  "Dissatisfied": "Dissatisfied",
+  "Very Dissatisfied": "Very Dissatisfied",
+  "Submit": "Submit",
+  "Microsoft reviews your feedback to improve our products, so please don't share any personal data or confidential/proprietary content.": "Microsoft reviews your feedback to improve our products, so please don't share any personal data or confidential/proprietary content.",
```

Comment:
Minor, but perhaps we should be stronger with the wording here and remove "please"; just:

> ...improve our products, so don't share any personal data...